run.py

```diff
@@ -15,6 +18,11 @@ def main():
 
     bot = Union(config=config)
 
+    try:
+        bot.load_extension("union.core.core")
+    except Exception as e:
+        print(f"Failed to load core cog.\n{e.__class__.__name__}: {e}")
```

Comment:
Original thought process went something along the lines of "well the bot will work even if the cog won't load" but since it is the core cog I guess the bot should stop if it fails to load.